pgxn/neon/file_cache.c

```diff
@@ -359,6 +389,17 @@ lfc_init(void)
 							lfc_change_limit_hook,
 							NULL);
 
+	DefineCustomBoolVariable("neon.file_cache_raw_block_device",
```

Comment:
I wonder if we really need this extra GUC?
Can we just add some prefix to `neon.file_cache_path`, i.e. "raw:..."